Bottom: 03db80a311947532a6e3dc529308bf26ae0a69d0
Top:    82d6dfaf114e6b7489b8e929cdf5b671646c329f
Author: Jon Smirl <jonsmirl@gmail.com>
Date:   2012-03-24 15:42:05 -0400

Refresh of devicetree

---

diff --git a/arch/arm/boot/dts/ea3131.dts b/arch/arm/boot/dts/ea3131.dts
index 5aaf2e7..020c063 100644
--- a/arch/arm/boot/dts/ea3131.dts
+++ b/arch/arm/boot/dts/ea3131.dts
@@ -29,6 +29,19 @@
 		reg = <0x30000000 0x4000000>;
 	};
 	ahb {
+		apb@13008000 {
+			i2c@1300a000 {
+				rtc@51 {
+					compatible = "nxp,pcf8563";
+					reg = <0x51>;
+				};
+				eeprom@52 {
+					compatible = "catalyst,24c32";
+					reg = <0x52>;
+					pagesize = <32>;
+				};
+			};
+		};
 		sdmmc@18000000 {
 			cd-gpio = <&gpio 0x87 0>;
 			power-gpio = <&gpio 0x45 0>;
diff --git a/arch/arm/boot/dts/lpc3131.dtsi b/arch/arm/boot/dts/lpc3131.dtsi
index ae6f45e..a04f2aa 100644
--- a/arch/arm/boot/dts/lpc3131.dtsi
+++ b/arch/arm/boot/dts/lpc3131.dtsi
@@ -34,7 +34,7 @@
 			interrupts = <29>;
 		};
 
-		apb0: apb@13000000 {
+		apb@13000000 {
 			compatible = "simple-bus";
 			#address-cells = <1>;
 			#size-cells = <1>;
@@ -72,28 +72,28 @@
 				reg = <0x13006000 0x200>;
 			};
 		};
-		apb1: apb@13008000 {
+		apb@13008000 {
 			compatible = "simple-bus";
 			#address-cells = <1>;
 			#size-cells = <1>;
 			ranges = <0x13008000 0x13008000 0x4000>;
 
-			timer0: timer@13008000 {
+			timer@13008000 {
 				compatible = "nxp,lpc31xx-timer";
 				reg = <0x13008000 0x400>;
 				interrupts = <5>;
 			};
-			timer1: timer@13008400 {
+			timer@13008400 {
 				compatible = "nxp,lpc31xx-timer";
 				reg = <0x13008400 0x400>;
 				interrupts = <6>;
 			};
-			timer2: timer@13008800 {
+			timer@13008800 {
 				compatible = "nxp,lpc31xx-timer";
 				reg = <0x13008800 0x400>;
 				interrupts = <7>;
 			};
-			timer3: timer@13008c00 {
+			timer@13008c00 {
 				compatible = "nxp,lpc31xx-timer";
 				reg = <0x13008c00 0x400>;
 				interrupts = <8>;
@@ -101,20 +101,23 @@
 			pwm@13009000 {
 				compatible = "nxp,lpc31xx-pwm";
 				reg = <0x13009000 0x200>;
-				interrupts = <11>;
 			};
-			i2c0: i2c@1300a000 {
-				compatible = "nxp,lpc31xx-i2c";
+			i2c@1300a000 {
+				#address-cells = <1>;
+				#size-cells = <0>;
+				compatible = "nxp,lpc31xx-i2c", "nxp,pnx-i2c";
 				reg = <0x1300a000 0x400>;
 				interrupts = <11>;
 			};
-			i2c1: i2c@1300a400 {
-				compatible = "nxp,lpc31xx-i2c";
+			i2c@1300a400 {
+				#address-cells = <1>;
+				#size-cells = <0>;
+				compatible = "nxp,lpc31xx-i2c", "nxp,pnx-i2c";
 				reg = <0x1300a400 0x400>;
 				interrupts = <12>;
 			};
 		};
-		apb2: apb@15000000 {
+		apb@15000000 {
 			compatible = "simple-bus";
 			#address-cells = <1>;
 			#size-cells = <1>;
@@ -141,7 +144,7 @@
 				interrupts = <18 19 20 21 22 23>;
 			};
 		};
-		apb3: apb@16000000 {
+		apb@16000000 {
 			compatible = "simple-bus";
 			#address-cells = <1>;
 			#size-cells = <1>;
@@ -153,7 +156,7 @@
 				interrupts = <13 14 15 16>;
 			};
 		};
-		apb4: apb@17000000 {
+		apb@17000000 {
 			compatible = "simple-bus";
 			#address-cells = <1>;
 			#size-cells = <1>;
